src/main/xerus/monstercat/api/APIConnection.kt

```diff
@@ -130,6 +128,14 @@ class APIConnection(vararg path: String): HTTPQuery<APIConnection>() {
 			return httpClient.execute(request, context)
 		}
 		
+		fun getRedirectedStreamURL(track: Track): String?{
```

Comment:
Please document this method

Reply:
dc9a4a86df0193f3e99839ac30b8b6972188867e